com.unity.multiplayer.mlapi/Runtime/AOI/ClientObjectMap.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using NUnit.Framework;
```

Comment:
why is this needed?

Reply:
Good catch, I can blame Rider here a bit; I added some debug code and Rider added the using's.